types/sandbox/prerequest.d.ts

```diff
@@ -89,32 +106,64 @@ declare interface Visualizer {
     clear(): void;
 }
 
+declare interface ExecutionInterface {
+    /**
+     * Stops the current request and its scripts from executing.
+     * @excludeFromTestScript
```

Comment:
Can we remove `@excludeFromTestScript`?

Reply:
I tried removing @excludeFromTestScript but actually we can not do it because we have two step process to generate types right.. if I try removing it from last step that is from prequqest.d.ts it actually goes away from intermediate index.d.ts as well causing it to be not used at all.
It does not show up in app autocomplete though, so end user experience remains unaffected, I understand we want to build the sandbox code irrespective of app behaviour but I hit a roadblock when trying to get rid of that from the prerequest.d.ts
<img width="1232" alt="Screenshot 2023-10-31 at 6 58 13 PM" src="https://github.com/postmanlabs/postman-sandbox/assets/37595282/e62935f6-009e-437f-8582-0d9f6fdfa7d5">